internal/qkcapi/api.go

```diff
@@ -785,8 +785,13 @@ func (e *EthBlockChainAPI) GetBalance(address common.Address, fullShardKey *hexu
 	return (*hexutil.Big)(balance), nil
 }
 
-func (e *EthBlockChainAPI) GetTransactionCount(address common.Address, fullShardKey *hexutil.Uint) (hexutil.Uint64, error) {
-	fullShardId, err := getFullShardId(fullShardKey)
+func (e *EthBlockChainAPI) GetTransactionCount(address common.Address, fullShardKey string) (hexutil.Uint64, error) {
```

Comment:
fullShardKey must be is optional, and the version is that hexutil.Uint64 don't support 0x00XX type.

Reply:
@mask-pp if change hexutil  ,i  agree

Reply:
ok